sklearn/cluster/k_means_.py

```diff
@@ -1522,7 +1525,7 @@ def partial_fit(self, X, y=None):
 
         """
 
-        X = check_array(X, accept_sparse="csr")
+        X = check_array(X, accept_sparse="csr", order="C")
```

Comment:
I think the docstring of `partial_fit` should be updated to mention that `X` will be copied if not C contiguous.